### PR TITLE
Conversations: only permit basic elements in unexpanded excerpts

### PR DESCRIPTION
**Pull request by bluefuton**
This is a quick experiment designed to address two problems:
- Blockquotes and images can sometimes dominate the excerpts shown in Conversations. For example, when everyone in a thread starts with a blockquote, it's hard to follow the conversation. Similarly, an image early on in a comment can mean you get very little to go on in the excerpt.
- The top of an image or video is sometimes visible just above the 'Read More' cutoff (see https://github.com/Automattic/wp-calypso/issues/17714).

29987378-3e393c48-8f68-11e7-88fd-a527f626cb97

I know we'll probably want to do this in a more performant manner if we stick with it (creating the stripped down excerpt on comment ingest, like we do for posts, perhaps). 

Give it a spin :)

### CODE REVIEW

**Comment by blowery** [comment]
I like it!

**Comment by samouri** [comment]
i also like this

**Comment by bluefuton** [comment]
Cool :) I shall refactor it to create the excerpt when we receive the comment.

**Comment by bluefuton** [comment]
The challenge is knowing when to display 'Read More' links. Sometimes we show the excerpt and it doesn't trigger `overflowY`, but the full comment would:

screen shot 2017-11-23 at 18 47 38

**Comment by bluefuton** [comment]
I think a CSS-only solution could work well here.

**Comment by bluefuton** [comment]
Same overflow conundrum with the pure CSS solution.

I've opted just to hide blockquotes in the excerpt for now, which is a solid improvement, and doesn't seem too affected by the 'Read more' problem.

**Comment by samouri** [comment]
code lgtm, and I'm okay with the behavior.
just need to test

**Comment by samouri** [comment]
testing.
